### PR TITLE
Run stylelint on all SCSS files

### PR DESCRIPTION
**Pull request by jryans**
This adds quotes so that globbing actually works as intended with `stylelint`.
Previously, we were relying on whatever a shell does with `**` style globs,
which meant that many files weren't picked up.

This means many files weren't being linted before, so the remaining commits will fix
those new errors.

### CODE REVIEW

File: res/css/views/dialogs/_DevtoolsDialog.scss [hunk 1]
```diff
@@ -160,34 +157,34 @@ limitations under the License.
             border-radius: 4px;
         }
 
-        &:after {
+        &::after {
             content: attr(data-tg-on);
-            background: #02C66F;
+            background: #02c66f;
```
Comment (turt2live) [inline res/css/views/dialogs/_DevtoolsDialog.scss]: this kind of change is a bit silly to enforce imo

File: res/css/views/dialogs/_DevtoolsDialog.scss [hunk 2]
```diff
@@ -135,6 +135,7 @@ limitations under the License.
     }
 }
 
+/* stylelint-disable no-descending-specificity */
```
Comment (turt2live) [inline res/css/views/dialogs/_DevtoolsDialog.scss]: A comment to explain why this is here would be nice

File: res/css/views/elements/_Field.scss [hunk 3]
```diff
@@ -144,6 +144,7 @@ limitations under the License.
     color: $greyed-fg-color;
 }
 
+/* stylelint-disable no-descending-specificity */
```
Comment (turt2live) [inline res/css/views/elements/_Field.scss]: comment

File: res/css/views/rooms/_EventTile.scss [hunk 4]
```diff
@@ -518,6 +513,7 @@ div.mx_EventTile_notSent.mx_EventTile_redacted .mx_UnknownBody {
 
 /* end of overrides */
 
+/* stylelint-disable no-descending-specificity */
```
Comment (turt2live) [inline res/css/views/rooms/_EventTile.scss]: comment

File: res/css/views/dialogs/_DevtoolsDialog.scss (hunk 1 above)
Comment (jryans) [reply res/css/views/dialogs/_DevtoolsDialog.scss]: Ah yeah, perhaps so... At the moment, we're mostly just using the `stylelint-config-standard` rule set, but we could tweak individual rules if we want to.

File: res/css/views/dialogs/_DevtoolsDialog.scss (hunk 2 above)
Comment (jryans) [reply res/css/views/dialogs/_DevtoolsDialog.scss]: Agreed, added comments.

**Pull request merged**